### PR TITLE
Fix lint for error and rollback handler

### PR DESCRIPTION
**Pull request by JakobMiksch**
- fixes previously broken linting for rollback and error handler
- harmonize linting rules with other workers
- perform linting with GitHub CI

### CODE REVIEW

**Comment by JakobMiksch** [comment]
@weskamm Would indeed be useful!

**Comment by JakobMiksch** [comment]
In case we move the eslint config to on location, we should also include "klips-api" and "geoserver-init"

File: worker/error-handler/package.json [hunk 1]
```diff
@@ -14,11 +14,12 @@
     "pino-pretty": "^9.1.1"
   },
   "devDependencies": {
-    "eslint": "^7.29.0"
+    "eslint": "^7.32.0",
+    "@babel/eslint-parser": "^7.19.1",
```
Comment (weskamm) [inline worker/error-handler/package.json:18]: this should not be required and can be removed

File: worker/error-handler/package.json [hunk 2]
```diff
@@ -14,11 +14,12 @@
...
+    "@babel/plugin-syntax-import-assertions": "^7.18.6"
```
Comment (weskamm) [inline worker/error-handler/package.json:19]: see above

File: worker/rollback-handler/.eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+env:
+  es2021: true
+  node: true
+parser: "@babel/eslint-parser"
+parserOptions:
+  requireConfigFile: false
+  ecmaVersion: 12
+  sourceType: "module"
+  babelOptions:
```
Comment (weskamm) [inline worker/rollback-handler/.eslintrc.yml:9]: this should not be required and can be removed

File: worker/rollback-handler/.eslintrc.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+env:
+  es2021: true
+  node: true
+parser: "@babel/eslint-parser"
```
Comment (weskamm) [inline worker/rollback-handler/.eslintrc.yml:4]: this should not be required and can be removed

File: worker/rollback-handler/package.json [hunk 5]
```diff
@@ -16,11 +16,12 @@
     "rabbitmq-worker": "git+https://github.com/klips-project/rabbitmq-worker.git"
   },
   "devDependencies": {
-    "eslint": "^7.29.0"
+    "eslint": "^7.32.0",
+    "@babel/eslint-parser": "^7.19.1",
```
Comment (weskamm) [inline worker/rollback-handler/package.json:20]: see above

File: worker/rollback-handler/package.json [hunk 6]
```diff
@@ -16,11 +16,12 @@
...
+    "@babel/plugin-syntax-import-assertions": "^7.18.6"
```
Comment (weskamm) [inline worker/rollback-handler/package.json:21]: see above

**Review by weskamm — CHANGES_REQUESTED** [review]
Please remove the usage of babel, as its not required here.
Otherwise, this looks good!

File: worker/rollback-handler/package.json (hunk 6 above)
Comment (JakobMiksch) [reply worker/rollback-handler/package.json]: done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)

File: worker/rollback-handler/package.json (hunk 5 above)
Comment (JakobMiksch) [reply worker/rollback-handler/package.json]: done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)

File: worker/rollback-handler/.eslintrc.yml (hunk 4 above)
Comment (JakobMiksch) [reply worker/rollback-handler/.eslintrc.yml]: done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)

File: worker/rollback-handler/.eslintrc.yml (hunk 3 above)
Comment (JakobMiksch) [reply worker/rollback-handler/.eslintrc.yml]: done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)

File: worker/error-handler/package.json (hunk 2 above)
Comment (JakobMiksch) [reply worker/error-handler/package.json]: done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)

File: worker/error-handler/package.json (hunk 1 above)
Comment (JakobMiksch) [reply worker/error-handler/package.json]: done with [548ca79](https://github.com/klips-project/klips-sdi/pull/131/commits/548ca79f0a2ffa74f3c33ad64e19f9a376258cdd)

**Comment by JakobMiksch** [comment]
@weskamm thanks for the review. 
Currently we still have multiple eslintrc files.
Shall we unit them to a single eslintrc file within this PR?

**Review by weskamm — APPROVED** [review]

**Comment by weskamm** [comment]
@JakobMiksch After having a detailed look i decided against that, as we have lots of different tools and things in here, so its better to have them separated

**Pull request merged**